javaslang/src/main/java/javaslang/collection/Seq.java

```diff
@@ -1153,4 +1153,6 @@ default boolean startsWith(Iterable<? extends T> that, int offset) {
     default Function1<Integer, T> withDefault(Function<? super Integer, ? extends T> defaultFunction) {
         return i -> (i >= 0 && i < length()) ? apply(i) : defaultFunction.apply(i);
     }
+
+    java.util.List<T> asJava();
```

Comment:
Oh , copy-paste error. I meant to move all the asJava() methods alphabetically (within the section above 'Adjusted return types')